docs/index.yaml

```diff
@@ -304,7 +331,8 @@ entries:
   - apiVersion: v1
     appVersion: "3.1"
     created: "2020-05-13T15:47:43.269858-04:00"
-    description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
```

Comment:
What happened here?

Reply:
Will check what changed

Reply:
i've tried this once more and the line break is the one causing all the changes.